tests/testthat/test_applyChangeMeta.R

```diff
@@ -311,3 +311,22 @@ test_that("sort value labels", {
   expect_equal(out, dfSAV$labels[1:3,])
 })
 
+
+test_that("Multiple meta data conflicts during value recoding", {
+  dat <- data.frame(v1 = factor(c("a", "b", "c", "d", "e")))
+  gads <- import_DF(dat)
+
+  expect_error(recodeGADS(gads, "v1", oldValues = c(1, 2), newValues = c(6, 6), existingMeta = "value_new"),
+               "Duplicated values in 'value_new' causing conflicting meta data in variable v1: 6. Use 'existingMeta' = 'drop' or 'ignore' to drop all related meta data.")

```

Comment:
Or maybe even `Set existingMeta to either \"drop\" or \"ignore\"` or `Use existingMeta = \"drop\" or existingMeta = \"ignore\"`

Reply:
I agree that `existingMeta` should be unquoted (is implemented)! If I come across this anywhere else in the package will fix. However, what is your opinion on occurences like
```
stop("Values in 'value_new' with existing meta data in variable ", var_name, ": ",
                                               paste(all_values, collapse = ", "))
``` 
In such cases the quotation marks indicate an argument name in my opinion (e.g., `'value_new'`. Would you still remove quotation marks here?

Regarding single vs. double quotation marks:
I have so far used single quotation marks for warnings, messages and errors throughout all my packages. I see why `\"` is more familiar for German users, however for an international audience (and all packages are documented in English anyway) single quotation marks make sense. Will leave this one as is.

Reply:
+1 regarding discussing this as part of our code convention meeting! Albeit I suppose this will be more than a one-time meeting but probably more like an ongoing discussion group 😄